### ISSUE TITLE
scope.isEquivalentTo(lastParent.getParent()) is false in processDeclarationsInPreviousSibling

### ISSUE THREAD

**Issue by mikeonslow** [KronicDeth/intellij-elixir#633]
# Version

4.7.0

# Exception

## Message

null

## Stacktrace

```
java.lang.AssertionError
	at org.elixir_lang.psi.impl.ElixirPsiImplUtil.processDeclarationsInPreviousSibling(ElixirPsiImplUtil.java:1679)
	at org.elixir_lang.psi.ElixirFile.processDeclarations(ElixirFile.java:45)
	at com.intellij.psi.util.PsiTreeUtil.treeWalkUp(PsiTreeUtil.java:992)
	at org.elixir_lang.psi.scope.variable.MultiResolve.resolveResultList(MultiResolve.java:90)
	at org.elixir_lang.psi.scope.variable.MultiResolve.addToResolveResultList(MultiResolve.java:207)
	at org.elixir_lang.psi.scope.variable.MultiResolve.addToResolveResultListIfMatchingName(MultiResolve.java:245)
	at org.elixir_lang.psi.scope.variable.MultiResolve.executeOnVariable(MultiResolve.java:155)
	at org.elixir_lang.psi.scope.Variable.execute(Variable.java:334)
	at org.elixir_lang.psi.scope.Variable.execute(Variable.java:107)
	at org.elixir_lang.psi.impl.ElixirPsiImplUtil.processDeclarations(ElixirPsiImplUtil.java:1643)
	at org.elixir_lang.psi.impl.ElixirUnmatchedMatchOperationImpl.processDeclarations(ElixirUnmatchedMatchOperationImpl.java:116)
	at com.intellij.psi.util.PsiTreeUtil.treeWalkUp(PsiTreeUtil.java:992)
	at org.elixir_lang.psi.scope.variable.MultiResolve.resolveResultList(MultiResolve.java:90)
	at org.elixir_lang.psi.scope.variable.MultiResolve.resolveResultList(MultiResolve.java:68)
	at org.elixir_lang.reference.Callable.multiResolve(Callable.java:644)
	at org.elixir_lang.reference.Callable.resolve(Callable.java:682)
	at com.intellij.codeInsight.TargetElementUtil.a(TargetElementUtil.java:371)
	at com.intellij.codeInsight.TargetElementUtil.b(TargetElementUtil.java:348)
	at com.intellij.codeInsight.TargetElementUtil.a(TargetElementUtil.java:215)
	at com.intellij.codeInsight.TargetElementUtil.findTargetElement(TargetElementUtil.java:184)
	at com.intellij.codeInsight.daemon.impl.IdentifierHighlighterPass.doCollectInformation(IdentifierHighlighterPass.java:91)
	at com.intellij.codeHighlighting.TextEditorHighlightingPass.collectInformation(TextEditorHighlightingPass.java:67)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass$1$1.run(PassExecutorService.java:444)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1177)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass$1.run(PassExecutorService.java:435)
	at com.intellij.openapi.progress.impl.CoreProgressManager.a(CoreProgressManager.java:446)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:392)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:54)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.a(PassExecutorService.java:432)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.run(PassExecutorService.java:408)
	at com.intellij.concurrency.JobLauncherImpl$VoidForkJoinTask$1.exec(JobLauncherImpl.java:206)
	at jsr166e.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at jsr166e.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:858)
	at jsr166e.ForkJoinPool.scan(ForkJoinPool.java:1687)
	at jsr166e.ForkJoinPool.runWorker(ForkJoinPool.java:1642)
	at jsr166e.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:108)

```

**Comment by KronicDeth** [comment]
> ```java
> assert scope.isEquivalentTo(lastParent.getParent());
> ```
> -- https://github.com/KronicDeth/intellij-elixir/blob/v4.7.0/src/org/elixir_lang/psi/impl/ElixirPsiImplUtil.java?utf8=%E2%9C%93#L1679

**Comment by KronicDeth** [comment]
Duplicate of #692.  Fixed by #756.  Wait for the next release to see the fix.
